app/task_routes.py

```diff
@@ -0,0 +1,122 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, jsonify, abort
+import datetime
+import os
+import requests
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+# create helper function to post message to Slack
+def slack_bot(slack_message):
+    path = "https://slack.com/api/chat.postMessage"
+    SLACK_API_KEY = os.environ.get("SLACK_API_KEY")
+    HEADERS = {"Authorization": SLACK_API_KEY}
+    channel_id ="C049QSML63U"
+    
+    query_params = {
+        "channel": channel_id, 
+        "text": slack_message
+        }
+
+    requests.post(path, params=query_params, headers=HEADERS)
+
+# validate ids for models
+def validate_id(cls, model_id):
```

Comment:
Nice work making this method generic enough that any class could use it to validate an id. I saw that you import this into goal_routes.py too. 

Since validate_id isn't specific to task or task_routes (which declaring it here kind of implies that it is), we can pull this method into a helper file and then import the method into the different routes files.